app/experimenter/nimbus-ui/src/components/AppLayoutSidebarLaunched/index.tsx

```diff
@@ -280,6 +282,23 @@ export const AppLayoutSidebarLaunched = ({
                 </DisabledItem>
               )}
               <SidebarActions {...{ experiment, refetch }} />
+              <p className="edit-divider position-relative small my-2">
+                <span className="position-relative bg-light pl-1 pr-2 text-muted">
+                  External Links
+                </span>
+              </p>
+              <LinkExternal
+                className="mx-1 my-2 nav-item d-block text-dark font-weight-semibold w-100"
+                href="https://experimenter.info"
+              >
+                <BookIcon className="mr-2" /> Experimenter Documentation
+              </LinkExternal>
+              <LinkExternal
+                className="mx-1 my-2 nav-item d-block text-dark font-weight-semibold w-100"
+                href="slack://channel?team=T027LFU12&id=CF94YGE03"
+              >
+                <SlackIcon /> #ask-experimenter
+              </LinkExternal>
```

Comment:
Since these links are identical between AppLayoutSidebarLaunched and AppLayourWithSidebar, maybe better to add these to the `<SidebarActions>` component which is shared between both?